sdmetrics/visualization.py

```diff
@@ -309,3 +426,67 @@ def get_column_plot(real_data, synthetic_data, column_name, plot_type=None):
     fig = _generate_column_plot(real_column, synthetic_column, plot_type)
 
     return fig
+
+
+def get_column_pair_plot(real_data, synthetic_data, column_names, plot_type=None):
+    """Return a plot of the real and synthetic data for a given column pair.
+
+    Args:
+        real_data (pandas.DataFrame):
+            The real table data.
+        synthetic_column (pandas.Dataframe):
+            The synthetic table data.
+        column_names (list[string]):
+            The names of the two columns to plot.
+        plot_type (str or None):
+            The plot to be used. Can choose between ``box``, ``heatmap``, ``scatter`` or ``None``.
+            If ``None` select between ``box``, ``heatmap`` or ``scatter`` depending on the data
+            that the column contains, ``scatter`` used for datetime and numerical values,
+            ``heatmap`` for categorical and ``box`` for a mix of both. Defaults to ``None``.
+
+    Returns:
+        plotly.graph_objects._figure.Figure
+    """
+    if len(column_names) != 2:
```

Comment:
Maybe we can put all the validation in a separate private method, like `_validate_input_plot()` or something else